### PR TITLE
Fix custom webhook request body format

### LINKED ISSUE #245: [BUG] Can't send custom webhooks
**Describe the bug**
Can't send custom webhooks that expect specific payload bodies.

1. Seems like we are wrapping all custom webhook payloads in { "Content": ... } which is specific to Chime.
2. And also seems like we are requiring a title and constructing the message as "$title\n\n$textDescription" which would break custom JSON payload to custom webhooks.

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description
- remove the forced "Content" key for custom webhooks, as the custom webhook server can require any format. User may provide json which contains customized format. 
- remove title for custom webook message, since newline breaks JSON format input. 
- Fix the issue that some unit test using junit4 not being picked up, by replacing with jupiter
- change common-utils branch name in CI to `main`
- suppport http as url protocol in webhook

### Issues Resolved
#245 
#232 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by zhongnansu** [comment]
CI is failing, but it passed locally. I also try re-run CI on HEAD of `dev` branch. It fails too

**Review by davidcui1225 — APPROVED** [review]

File: notifications/spi/src/test/kotlin/org/opensearch/notifications/spi/CustomWebhookDestinationTests.kt [hunk 1]
```diff
@@ -36,8 +36,9 @@ import org.apache.http.entity.StringEntity
 import org.apache.http.impl.client.CloseableHttpClient
 import org.apache.http.message.BasicStatusLine
 import org.easymock.EasyMock
-import org.junit.Test
```
Comment (davidcui1225) [inline notifications/spi/src/test/kotlin/org/opensearch/notifications/spi/CustomWebhookDestinationTests.kt:39]: How come we changed the `Test` import?

**Review by davidcui1225 — APPROVED** [review]
just one question

File: notifications/spi/src/test/kotlin/org/opensearch/notifications/spi/CustomWebhookDestinationTests.kt (hunk 1 above)
Comment (zhongnansu) [reply notifications/spi/src/test/kotlin/org/opensearch/notifications/spi/CustomWebhookDestinationTests.kt:39]: we are using junit5, but this one is still junit4, it doesn't throw error, but actually the tests are not picked up. We should use junit 5 anywhere

**Pull request merged**